.gitignore

```diff
@@ -1 +1,23 @@
-node_modules
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
Noted with Thanks. 
However, neither of us has own logo to add in ReadMe file.